moveit_setup_assistant/moveit_setup_framework/src/urdf_config.cpp

```diff
@@ -92,32 +92,45 @@ void URDFConfig::loadFromPath(const std::filesystem::path& urdf_file_path, const
 
 void URDFConfig::setPackageName()
 {
-  bool package_found = extractPackageNameFromPath(urdf_path_, urdf_pkg_name_, urdf_pkg_relative_path_);
-  if (!package_found)
-  {
-    urdf_pkg_name_ = "";
-    urdf_pkg_relative_path_ = urdf_path_;  // just the absolute path
-  }
-  // Check that ROS can find the package
-  const std::filesystem::path robot_desc_pkg_path = getSharePath(urdf_pkg_name_);
+  // Reset to defaults: no package name, relative path is set to absolute path
```

Comment:
These changes silence the warning if there is no package name associated with the file.